lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Here $\\Omega$ is some space of all allowed models. As we normally do for parameter inference, we'll work with a simplified version:\n",
+    "\n",
+    "$P(H|D)\\propto P(D|H)P(H)$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "$P(H)$ is a prior on the model, and\n",
+    "\n",
+    "$P(D|H)=\\int P(D|\\theta,H) \\, P(\\theta|H) d\\theta$ \n",
+    "\n",
+    "is the **evidence** - the normalizing denominator in Bayesian parameter inference (also known as the **fully marginalized likelihood**)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Ideally, we would compare models by looking at\n",
+    "\n",
+    "$\\frac{P(H_2|D)}{P(H_1|D)}=\\frac{P(D|H_2)\\,P(H_2)}{P(D|H_1)\\,P(H_1)}$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "General difficulties in computing the terms in this ratio:\n",
+    "* Assigning meaningful priors to models\n",
+    "* Assigning meaningful priors to parameters\n",
+    "* Calculating the evidence integral"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "**Exercise: \"non-informative\" parameter priors and the evidence**\n",
+    "\n",
+    "Say we have a model with 1 parameter, $\\theta$, and a likelihood that works out to be a unit width Gaussian centered on $\\theta=0$ with peak value $\\mathcal{L}_{\\rm max}$.\n",
+    "\n",
+    "For each of the priors on $\\theta$ below, (a) sketch the likelihood and prior as a function of theta, (b) calculate the evidence (well enough to comment about the relative differences, anyway).\n",
+    "1. $P(\\theta)$ uniform on $[-1,+1]$\n",
+    "2. $P(\\theta)$ uniform on $[-3,+3]$\n",
+    "3. $P(\\theta)$ uniform on $[-100,+100]$\n",
+    "4. $P(\\theta)$ uniform on $[-\\infty,+\\infty]$\n",
+    "5. $P(\\theta)$ uniform on $[+3,+5]$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "**Evidence exercise: notes**\n",
+    "\n",
+    "1) Models that are less prescriptive (in terms of their priors) are penalized in the evidence. This is a feature, although it means we need to put real thought into those priors.\n",
+    "\n",
+    "2) The evidence can be made arbitrarily small by increasing the prior volume: comparing evidences is more conservative than focusing on the goodness of fit ($L_{\\rm max}$) alone. \n",
+    "\n",
+    "3) The evidence is linearly sensitive to prior volume ($f$), but exponentially sensitive to goodness of fit ($L_{\\rm max} \\propto e^{-\\chi^2_{\\rm min}/2}$). It's still a likelihood, after all."
```

Comment:
What is the $f$ here?

Reply:
Ah - an area fraction used in a graphical solution to the exercise. I'll remove this reference but include some example sketches.